--- spec/addressable/uri_spec_orig.rb
+++ spec/addressable/uri_spec.rb
@@ -401,9 +401,9 @@
     expect(@uri.normalized_host).to eq("example.com")
   end
 
-  it "returns 'com' for #tld" do
-    expect(@uri.tld).to eq("com")
-  end
+  # it "returns 'com' for #tld" do
+  #   expect(@uri.tld).to eq("com")
+  # end
 
   it "returns 'user:password@example.com:8080' for #authority" do
     expect(@uri.authority).to eq("user:password@example.com:8080")
@@ -2393,9 +2393,9 @@
     expect(@uri.origin).to eq('http://example.com')
   end
 
-  it "should have a tld of 'com'" do
-    expect(@uri.tld).to eq('com')
-  end
+  # it "should have a tld of 'com'" do
+  #   expect(@uri.tld).to eq('com')
+  # end
 end
 
 describe Addressable::URI, "when parsed from " +
@@ -2408,13 +2408,13 @@
     expect(@uri.origin).to eq('http://www.example.co.uk')
   end
 
-  it "should have a tld of 'co.uk'" do
-    expect(@uri.tld).to eq('co.uk')
-  end
-
-  it "should have a domain of 'example.co.uk'" do
-    expect(@uri.domain).to eq('example.co.uk')
-  end
+  # it "should have a tld of 'co.uk'" do
+  #   expect(@uri.tld).to eq('co.uk')
+  # end
+
+  # it "should have a domain of 'example.co.uk'" do
+  #   expect(@uri.domain).to eq('example.co.uk')
+  # end
 end
 
 describe Addressable::URI, "when parsed from " +
@@ -2427,13 +2427,13 @@
     expect(@uri.origin).to eq('http://sub_domain.blogspot.com')
   end
 
-  it "should have a tld of 'com'" do
-    expect(@uri.tld).to eq('com')
-  end
-
-  it "should have a domain of 'blogspot.com'" do
-    expect(@uri.domain).to eq('blogspot.com')
-  end
+  # it "should have a tld of 'com'" do
+  #   expect(@uri.tld).to eq('com')
+  # end
+
+  # it "should have a domain of 'blogspot.com'" do
+  #   expect(@uri.domain).to eq('blogspot.com')
+  # end
 end
 
 describe Addressable::URI, "when parsed from " +

