--- Gemfile
+++ Gemfile
@@ -14,13 +14,11 @@
 end
 
 group :test, :development do
-  gem 'rake', '> 10.0', '< 12'
+  gem 'rake', '> 10.0', '< 12.1'
   gem 'simplecov', :require => false
   gem 'coveralls', :require => false, :platforms => [
     :ruby_20, :ruby_21, :ruby_22, :ruby_23
   ]
-  # Used to test compatibility.
-  gem 'rack-mount', git: 'https://github.com/sporkmonger/rack-mount.git', require: 'rack/mount'
 
   if RUBY_VERSION.start_with?('2.0', '2.1')
     gem 'rack', '< 2', :require => false

